client-templates/nexus/.env.sample

```diff
@@ -0,0 +1,24 @@
+# your unique broker identifier
+BROKER_TOKEN=<broker-token>
+
+# The URL to your Nexus Repository Manager
+# If not using basic auth this will only be "<your.nexus.hostname>/repository"
+NEXUS_URL=<username>:<password>@<your.nexus.hostname>/repository
+
+# The URL of the Snyk broker server
+BROKER_SERVER_URL=https://broker.snyk.io
+
+# the fine detail accept rules that allow Snyk to make API requests to your
+# nexus instance
+ACCEPT=accept.json
+
+# The path for the broker's internal healthcheck URL. Must start with a '/'.
+BROKER_HEALTHCHECK_PATH=/healthcheck
+
+# Provide RES_BODY_URL_SUB with the URL of the nexus without credentials and http protocol
+# This URL substitution is required for NPM integration
+# RES_BODY_URL_SUB=http://<your.nexus.hostname>/repository
```

Comment:
not sure about this - copied from artifactory

Reply:
probably good enough as an example - we need to test nexus npm responses. Artifactory was a little strange and required us to put "http" instead of what we expected ("https").